sdks/python/apache_beam/io/gcp/datastore/v1new/datastoreio_test.py

```diff
@@ -49,8 +49,20 @@
   client = None
 
 
-# TODO(https://github.com/apache/beam/issues/25625)
-# remove this FakeMutation and replace it with proto.Message class.
+# used for internal testing only
+class ByteSize():
```

Comment:
Name this class as Fake[ClassNameThatYouAreFaking], for example: FakeMessage.

Reply:
Changed it to FakeByteSize()